grafana/grafana
# InfluxDB: interpolate all variables in queries for dash->explore (for influxql)

**What this PR does / why we need it**:
This PR ensures that all variables are interpolated in queries when going from the dashboard to explore (for influxql). 

Previously we only interpolated query, measurement, policy and tags.

**How to test**
1. Create a dashboard panel for influxql
2. Edit the panel, adding variables to all parts of the query
3. Save, go back to the dash
4. Hover over the panel and click x to render the query in explore
5. All variables should be interpolated and the query should succeed
